src/main/java/org/openpnp/machine/reference/vision/wizards/ReferenceBottomVisionPartConfigurationWizard.java

```diff
@@ -133,6 +131,14 @@ private void testAlignment() throws Exception {
 
         // position the part over camera center
         Location cameraLocation = VisionUtils.getBottomVisionCamera().getLocation();
+		
+		if(alignmentOffset.getPreRotated()) {
+			if(Math.abs(alignmentOffset.getLocation().convertToUnits(LengthUnit.Millimeters).getLinearDistanceTo(0.,0.))>19.999) {
```

Comment:
How did you come up with 19.999? Is that based on something specific, or is it just a "large" value?